woocommerce/woocommerce-android
# Display date instead of time on order creation

Closes: #6389


### Description

This PR updates the `updateOrder` function to show the date of order creation from the start in the Order details as opposed to the time only on the first day. 

### Testing instructions


- Go to the Orders screen
- Tap on the + button
- tap on Create Order
- Make sure the current date is displayed at the top of the screen as opposed to a time stamp

### Images/gif


| Before | After |
| ---- | ---- |
| <img src="https://user-images.githubusercontent.com/30724184/167927478-b0bf4f9d-e934-40f5-94a2-42ab821cfe48.png" width="350"/> | <img src="https://user-images.githubusercontent.com/30724184/167925321-b558f536-351c-4633-9039-c5de9e601718.png" width="350"/> |
| <img src="https://user-images.githubusercontent.com/30724184/167927549-12f62ed1-69ff-4b07-bf2b-9cc0a57b75c1.png" width="350"/> | <img src="https://user-images.githubusercontent.com/30724184/167925367-594eaa9c-74db-4116-b38f-d91400389318.png" width="350"/> |
